src/pages/LogView.tsx

```diff
@@ -8,9 +8,9 @@ interface LogViewProps {
 }
 
 const LogView: React.FC<LogViewProps> = ({ logType }) => {
-  const { hasLogs } = useLogContext();
+  const { hasLogs, logsAreEmpty } = useLogContext();
 
-  return hasLogs ? (
+  return hasLogs || logsAreEmpty ? (
     <LogWindow isUploadedLog={false} logType={logType} />
   ) : (
     <LoadingPage logType={logType} />
```

Comment:
Hmm it's somewhat confusing to me how there is `hasLogs` and `logsAreEmpty`, since they sound like they represent the same thing. One thing I think might be more intuitive would be to have a `loading` boolean that starts as true and gets set to false in `INGEST_LOGS`. That way, you could flip this condition:

```
  const { loading } = useLogContext();

  return loading ? (
    <LoadingPage logType={logType} />
  ) : (
    <LogWindow isUploadedLog={false} logType={logType} />
  );
```

I removed `logsAreEmpty` and tried it this way and I think it worked with your existing changes! Pls feel free to DM me if this doesn't make sense haha 😅